src/components/RichTextEditor/serializer/markdown/deserializer/plugin.ts

```diff
@@ -1,9 +1,9 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { MarkdownAstNode, OptionType } from '../types';
+import { MarkdownAstNode, PartialOptionType } from '../types';
 import deserialize from './deserialize';
 
-export default function plugin(opts?: OptionType) {
+export default function plugin(opts?: PartialOptionType) {
```

Comment:
```suggestion
export default function plugin(options?: PartialOptionType) {
```
Same for the others files 🙈

Reply:
Updated

Reply:
Updated